cms/templates/js/xblock-access-editor.underscore

```diff
@@ -0,0 +1,5 @@
+<div class="incontext-editor-action-wrapper">
+    <a href="" class="unit-container visibility-button" title="<%- gettext('Access Settings') %>">
```

Comment:
@cahrens does it help to add a # instead of being empty? but you are right if it's acting as a button then it should be a button.